src/parcel.proptypes.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types'
+import Parcel from './parcel'
+
+Parcel.propTypes = {
+  // Required
+  config: PropTypes.oneOfType([
+    // A config object
+    PropTypes.shape({
+      bootstrap: PropTypes.func.isRequired,
+      mount: PropTypes.func.isRequired,
+      unmount: PropTypes.func.isRequired,
+      update: PropTypes.func,
+      name: PropTypes.string,
+    }).isRequired,
+
+    // a loading function
+    PropTypes.func,
+  ]),
+
+  // Optional
+  mountParcel: PropTypes.func.isRequired, // Required if it is not available from SingleSpaContext (i.e., when parcel is not rendered by a single-spa application)
```

Comment:
I like seeing your adoption of PropTypes. Makes me want to do it more in my own projects